extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -46,7 +46,7 @@ private static function renderIcon( $media ) {
 					'width' => $scaledSize['width'],
 					'title' => $media['title'],
 					'link' => $media['link'],
-					'caption' => $media['caption']
+					'caption' => $media['caption'] ?? ''
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "caption" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)